### PR TITLE
[2주차] - unet/experiment.html/김민영

### PR DESCRIPTION
**Pull request by minyoungci**
1. experiment.html 477 - " Crop the target mask to the size of the logits. Size of the logits will be smaller if we dont use padding in convolutional layers in the U-Net. " 

logits 는 그냥 logits으로 뒀고, mask도 mask로 영어 사용하여 번역했습니다. 

2. experiment.html 572 - "New line in the console"

'콘솔에서의 새로운 라인' 으로 해석하는게 맞을까요?

### CODE REVIEW

**Comment by kimdahyeon977** [comment]
logits 은 로짓으로 하는게 좋지 않을까요 ?
저는 mask는 그냥 마스크로 했습니다 ㅎㅎ

콘솔의 새로운 줄이라고 하는 건 어떨까요 ??
수고하셨습니다!
